test/test_cromwell_workflow.sh

```diff
@@ -2,26 +2,27 @@
 
 set -e
 
-CROMWELL_USER=$1
-CROMWELL_PASSWORD=$2
-CROMWELL_URL=$3
-INPUTS_JSON=$4
-WDL_FILE=$5
-OPTIONS_FILE=$6
-DEPENDENCIES_JSON=$7
+CROMWELL_KEY_FILE=$1
+CROMWELL_URL=$2
+INPUTS_JSON=$3
+WDL_FILE=$4
+OPTIONS_FILE=$5
+DEPENDENCIES_JSON=$6
+COLLECTION=$7
 # Read list of dependency files into an array to pass to cromwell-tools
 DEPENDENCIES_LIST=$(cat ${DEPENDENCIES_JSON} | python3 -c "import json,sys;obj=json.load(sys.stdin);print(' '.join(obj.values()));")
 echo ${DEPENDENCIES_LIST[@]}
 
 echo "Starting workflow."
+
 WORKFLOW_HASH=$(cromwell-tools submit \
-  --username "${CROMWELL_USER}" \
-  --password "${CROMWELL_PASSWORD}" \
+  --service-account-key "${CROMWELL_KEY_FILE}" \
   --url "${CROMWELL_URL}" \
   --inputs_files "${INPUTS_JSON}" \
   --wdl-file "${WDL_FILE}" \
   --options-file "${OPTIONS_FILE}" \
   --deps-file ${DEPENDENCIES_LIST[@]} \
+  --collectiuon-name ${COLLECTION} \
```

Comment:
Can you investigate if this is being called? There's a spelling error here that makes me question if this does anything.

Reply:
That spelling error has been corrected, but there are still some bugs I will change this from draft once everything is working properly

Reply:
Whoa, I have never seen draft PRs before. Sorry for pulling the trigger too early on the review!